syncd/syncd.cpp

```diff
@@ -346,6 +348,8 @@ void remove_rid_and_vid_from_local(
 
     local_rid_to_vid.erase(rid);
     local_vid_to_rid.erase(vid);
+
+    g_removed_rid_to_vid[rid] = vid;
```

Comment:
>g_removed_rid_to_vid [](start = 4, length = 20)

I have concern about the long run. If the switch is running health for long, `g_removed_rid_to_vid` will accumulate all the history `rid`. There is no way to release the memory.

Reply:
yes there is - apply view will clear that, until restart, yes, it can accumulate, but this is temporary solution, to move things going on 201911 branch, with fdb notification for removed bridge port. Im also working on fix on master. 
Do you have any proposal for making this better? Note that those oids will be checked in fdb notification, and we don't know how long it takes to process notifications, and how many notifications there can be to process.
please refer to in depth problem description on "Re: PRs for FDB fix issue" internal thread